### PR TITLE
Update: Prevent changes to the report while it is running

### PR DESCRIPTION
**Pull request by mattgraham1995**
## Description
While a report is running, changing the request by interacting with the report builder was causing a disconnect between the visualization/request of the report and the response data. 

## Proposed Changes

- Prevent use of the report builder components while the report is running
- Disable filter icons and checkboxes for metric and dimension selectors while report is running, but still allow navigation through them
- Completely disable all clicks on the filter collection while report is running
- Disable logical table selector dropdown
- Give disabled components a slightly darker gray background

## Screenshots
Screen Shot 2019-06-24 at 6 36 52 PM

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/reports/tests/acceptance/navi-report-test.js [hunk 1]
```diff
@@ -1705,12 +1717,16 @@ module('Acceptance | Navi Report', function(hooks) {
   });
 
   test('Cancel Report', async function(assert) {
+    assert.expect(30);
     //Slow down mock
```
Comment (kevinhinterlong) [inline packages/reports/tests/acceptance/navi-report-test.js]: Remove this?

File: packages/reports/app/styles/navi-reports/components/report-builder.less [hunk 2]
```diff
@@ -23,6 +23,14 @@
     padding: 5px 10px;
   }
 
+  &__container--disabled {
+    background: @denali-gray-300;
```
Comment (jkusa) [inline packages/reports/app/styles/navi-reports/components/report-builder.less]: Can we set `cursor: not-allowed;`;

Comment (cythrawll) [reply packages/reports/app/styles/navi-reports/components/report-builder.less]: is there a `cursor: banished-to-void-realm;`?

Comment (jkusa) [reply packages/reports/app/styles/navi-reports/components/report-builder.less]: And `pointer-events: none;`

Comment (mattgraham1995) [reply packages/reports/app/styles/navi-reports/components/report-builder.less]: What do you guys think about showing the selected metrics and dimensions when the report is run? Then when the report finishes running, the metric and dimension selectors will go back to whatever state they were in before running.

**Pull request merged**